slides/02-data-budget.qmd

```diff
@@ -24,59 +24,68 @@ knitr:
 
 ##  {background-image="https://media.giphy.com/media/Lr3UeH9tYu3qJtsSUg/giphy.gif" background-size="40%"}
 
-## Data on tree frog hatching
+## Data on Chicago taxi trips
 
-![](images/Hatching-process.jpg)
+![](images/taxi.png)
 
-## Data on tree frog hatching
+::: footer
+Credit: <https://unsplash.com/photos/7_r85l4eht8>
+:::
+
+## Data on Chicago taxi trips
 
--   Red-eyed tree frog embryos can hatch earlier than their normal \~7 days if they detect potential predator threat!
--   Type `?stacks::tree_frogs` to learn more about this dataset, including references.
--   We are using a slightly modified version from stacks.
+-   The city of Chicago releases anonymized trip-level data on taxi trips in the city.
+-   We pulled a sample of 10,000 rides occurring in early 2022.
+-   Type `?modeldatatoo::data_taxi()` to learn more about this dataset, including references.
+-   We are using a slightly modified version from the modeldatatoo data.
 
 ```{r}
 library(tidymodels)
+library(modeldatatoo)
 
-data("tree_frogs", package = "stacks")
-tree_frogs <- tree_frogs %>%
-  mutate(t_o_d = factor(t_o_d),
-         age = age / 86400) %>%
-  filter(!is.na(latency)) %>%
-  select(-c(clutch, hatched))
+taxi <- data_taxi() %>%
+  mutate(month = factor(month, levels = c("Jan", "Feb", "Mar", "Apr"))) %>% 
+  select(-c(id, duration, fare, tolls, extras, total_cost, payment_type)) %>% 
+  drop_na()
 ```
 
-## Data on tree frog hatching
+## Data on Chicago taxi trips
 
 ::: columns
 ::: {.column width="60%"}
--   `N = 572`
--   A numeric outcome, `latency`
--   4 other variables
-    -   `treatment`, `reflex`, and `t_o_d` are **nominal** predictors
-    -   `age` is a **numeric** predictor
+-   `N = 10,000`
+-   A nominal outcome, `tip`, with levels `"yes"` and `"no"`
+-   6 other variables
+    -   `company`, `local`, and `dow`, and `month` are **nominal** predictors
+    -   `distance` and `hours` are **numeric** predictors
 :::
 
 ::: {.column width="40%"}
-![](images/Ac_2tads.jpg)
+![](images/taxi.png)
```

Comment:
Surely not as cute as the tree frog graphics!

Reply:
There is also this cute little svg https://www.svgrepo.com/svg/8322/taxi, I could make the wheels spin if we would like

![taxi-svgrepo-com](https://github.com/tidymodels/workshops/assets/14034784/29d1da55-7bda-423b-9bd9-c60c9ed97b6b)

Reply:
haha i'm with it! i'd say let's do it

Reply:
I addeI it as an separate issue https://github.com/tidymodels/workshops/issues/111, so I can come back to it 😄

Reply:
Love it! 🚕

Reply:
Added! [7517077](https://github.com/tidymodels/workshops/pull/108/commits/751707709b6b6da61cc77a46aaa6f9ccfa3ae368)